run-reports.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+export TASKCLUSTER_ROOT_URL=https://firefox-ci-tc.services.mozilla.com/
+export REPORT_SCHEDULER_ID="smoketest"
+export REPORT_PREFIX=https://github.com/taskcluster/mozilla-history/blob/master/WorkerVersions/
+
+# needs to be set in the environment:
+# TASKCLUSTER_CLIENT_ID
+# TASKCLUSTER_ACCESS_TOKEN
+
+set -euo pipefail
+
+git pull
+
+echo "generating audit report"
+./mozilla-history
+
+git add .
```

Comment:
looks like mozilla-history binary is not in `.gitignore` - i'm not sure how this script avoided committing it until now? I think it needs to be added.

Reply:
Note, if `mozilla-history` isn't in the same directory, and is just somewhere in the `PATH` then this wouldn't be needed. But the reason I think it is needed is because you have `./mozilla-history` above, suggesting it is in this directory. But happy not to have it in that directory, not need the `.gitignore` and to just run `mozilla-history` instead of `./mozilla-history` above.